app/models/piece.rb

```diff
@@ -1,7 +1,7 @@
-require 'pry'
 class Piece < ActiveRecord::Base
   belongs_to :user
   belongs_to :game
+  has_many :types
```

Comment:
@CodeFlowed Oops...that's a line I was trying to debug and added in with a typo... mistakenly forgot to remove it. Thanks...I'll remove that and merge it!